app/src/main/java/org/oppia/android/app/administratorcontrols/LoadProfileEditListener.kt

```diff
@@ -0,0 +1,7 @@
+package org.oppia.android.app.administratorcontrols
+
+/** Listener for when an activity should load [ProfileEditFragment]. */
+interface LoadProfileEditListener {
+  /** This method is called when the fragment of [ProfileEditFragment] is inflated. */
```

Comment:
Method KDocs should describe what the method does, not when it's called (since that's tied just to the current implementation, and implementations change--it's generally more useful to describe what what and how of a method in its documentation so that people understand when and why to call it).

Reply:
done